modules/mailer/mailer.go

```diff
@@ -155,7 +155,7 @@ func processMailQueue() {
 			if !open {
 				var err error
 				if conn, err = dialer.Dial(); err != nil {
-					panic(err)
+					log.Error(4, "Fail to connect to SMTP server: %v", err)
```

Comment:
Perhaps we should check whether this is a temporal or a permanent error, reenqueue the message and try again later?